src/views/ManageList.jsx

```diff
@@ -1,7 +1,80 @@
-export function ManageList() {
+import { addItem } from '../api/firebase';
+import { useRef } from 'react';
+
+export function ManageList({ listPath }) {
+	const formRef = useRef();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		const formData = new FormData(e.target);
+		const itemName = formData.get('itemName');
+		const daysUntilNextPurchase = parseInt(
+			formData.get('daysUntilNextPurchase'),
+		);
+
+		try {
+			await addItem(listPath, {
+				itemName,
+				daysUntilNextPurchase,
+			});
+
+			alert('Item saved successfully');
+		} catch (error) {
+			alert('There was a problem');
+		}
+
+		formRef.current.reset();
```

Comment:
I wonder whether this `reset()` should be inside of the `try` branch of the `try ... catch ...`. If there is an error that means that the item wasn't saved, so we probably don't want to clear the form on people (in case they want to try again?)

Reply:
This makes sense, clearing a form after a successful submit is expected behavior, not clearing it would clearly indicate there was a problem. I added this change. 

I also modified the `catch` block to display a more detailed error message.